src/algebra/big_operators.lean

```diff
@@ -264,6 +264,18 @@ lemma prod_range_succ' (f : ℕ → β) :
 | (n + 1) := by rw [prod_range_succ (λ m, f (nat.succ m)), mul_assoc, ← prod_range_succ'];
                  exact prod_range_succ _ _
 
+@[to_additive finset.sum_range_zero]
+lemma prod_range_zero (f : ℕ → β) :
+ (range 0).prod f = 1 := by {rw[range_zero,prod_empty]}
```

Comment:
Please put the proof on a new line, remove the `{  }` and put a space after the `,`.

Reply:
Done.